crates/web-sys/webidls/enabled/HTMLInputElement.webidl

```diff
@@ -135,6 +135,8 @@ interface HTMLInputElement : HTMLElement {
   [Throws]
   undefined setSelectionRange(unsigned long start, unsigned long end, optional DOMString direction);
 
+  undefined showPicker();
```

Comment:
```suggestion
  [Throws]
  undefined showPicker();
```

This needs to be marked with `[Throws]` so that it'll return a `Result` in Rust.

Reply:
Thank you for the review! I've fixed the code and added documentation about it at the place where I read about how to add an interface :)